chucks-pick3-web/angular-app/src/app/shared/components/app.component/app.component.html

```diff
@@ -5,13 +5,13 @@
 </div>
 
 
-  <div class= "splash-container" *ngIf="showSplashPage === true">
-    <div class="splash-logo-container">
-      <img src="assets/images/chucksPickThreeLogo.png" alt="logo" width="300px" height="300px"/>
-    </div>
-    <div class=" link-container">
-      <span><a [routerLink]="'/home'" (click)="showSplashView(false)">LEARN MORE</a></span>
-    </div>
+<div class="splash-container" *ngIf="showSplashPage === true">
+  <div class="splash-logo-container">
+    <img src="src/app/assets/images/chucksPickThreeLogo.png" alt="logo" width="300px" height="300px"/>
```

Comment:
Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

Reply:
reverted to assets/images